packages/imperative/src/config/cmd/auto-init/handlers/BaseAutoInitHandler.ts

```diff
@@ -0,0 +1,170 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandHandler, IHandlerParameters, ICommandArguments } from "../../../../../../cmd";
+import { ISession, ConnectionPropsForSessCfg, Session, AbstractSession } from "../../../../../../rest";
+import { ConfigConstants, ConfigSchema, IConfig } from "../../../../../../config";
+import { diff } from "jest-diff";
+import stripAnsi from "strip-ansi";
+import * as open from "open";
+import * as JSONC from "comment-json";
+import * as lodash from "lodash";
+import { ImperativeConfig, TextUtils } from "../../../../../../utilities";
+
+/**
+ * This class is used by the auto init command handler as the base class for its implementation.
+ */
+export abstract class BaseAutoInitHandler implements ICommandHandler {
```

Comment:
I was wondering if BaseAutoInitHandler and BaseAuthHandler should've shared a common handler, but talked myself out of it.
While they do have some basic structure + abstract properties and function in common, it may be more trouble than what's worth.
We may reconsider this if/when we get a 3rd BaseHandler that required the same abstract properties and functions.